Alertmanager: Strict initialization (#10785)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -alertmanager.enable-state-cleanup
[enable_state_cleanup: <boolean> | default = true]

# (experimental) Skip initializing Alertmanagers for tenants without a
# non-default, non-empty configuration. For Grafana Alertmanager tenants,

> # configurations not marked as 'promoted' will also be skipped.

Reviewer: ```suggestion
# configurations not marked as 'promoted' are also skipped.
```
